metadata/society/schools_prek_to_12.md

```diff
@@ -0,0 +1,80 @@
+# Title
+
+Utah Schools Pre K to 12
+
+## ID
+
+0bd3da1d-be75-4364-9958-2d73e46e818e
+
+## Brief Summary
+
+Points dataset of locations of preschool, K-12, and other schools in Utah.
+
+## Summary
+
+This dataset contains point locations for preschool and K-12 schools in the state of Utah as of the 2019-2020 school year. Colleges, universities, and adult vocational training facilities are not included in the dataset.
```

Comment:
I think we can say that the data is current as of the 2023-2024 school year. I was given all the schools that we were missing, however, I was unable to update the demographics side. Maybe we keep that as a UGRC secret.

Reply:
So the points are current as of 2024 but the demographics data is as of 2019-2020?  I can make a note about it in the metadata and direct people to USBE if they are looking for current demographic info. In the past, have we updated both the points and the demographics data every year with USBE's annual report?

Reply:
Also, is there a place where we maintain previous years' school data for those interested in analyzing change over time? I didn't see any other school layers on the index, but wanted to be sure. @ZachBeck

Reply:
Yep, current as of 2024, and demographics as of 2019-2020. The creation of the dataset by WFRC back in 2020 is the only time the school locations and demographics have been updated together. Adding demographics to the schools was a bad call because there was never a real join key between the two datasets and was kind of fudged IIRC

Reply:
and no, we haven't archived past years or anything like that. Unfortunately Schools has been updated when folks have notified us of missing schools.

Reply:
Okay, good to know. I'll get that info added in. Thank you!